sys/dev/atkbdc/atkbdc.c

```diff
@@ -80,6 +83,9 @@
 #define write_command(k, d)	\
 			(bus_space_write_1((k)->iot, (k)->ioh1, 0, (d)))
 
+#define ATKBD_DELAY(x)	(atkbd_short_delay ? (x) : (x) * MILISECOND_MULTIPLIER)
+#define RESET_DELAY(x)	(atkbd_short_delay ? (x) * MILISECOND_MULTIPLIER : (x) * MILISECOND_MULTIPLIER / RESET_DELAY_DIVISION_FACTOR)
```

Comment:
with long macro body we use \ to continue on next line and next line will be indented